i18nify-data/country-zipcode/Description.md

```diff
@@ -0,0 +1,8 @@
+| Title         | ---                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Shouldn't we rename these files to README.md ? 
That way it'll be viewable on pages like these 
https://github.com/razorpay/i18nify/tree/re-arch/i18nify-multiple-langs/i18nify-data/country-zipcode

Reply:
Even the readme name can be put, but just because it have only descriptions of the different attributes of the data, we named it as Description. instead of readme.